packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.stories.js

```diff
@@ -20,35 +34,34 @@ export default {
   }
 }
 
-const background = {
-  dark: 'dark-1',
-  light: 'light-1'
-}
+const store = mockStore()
 
-export const LightTheme = ({ content }) => {
-  return (
-    <Grommet
-      background={background}
-      theme={zooTheme}
-      themeMode='light'
-    >
-      <Box height='500px' width='large'>
-        <SingleTextViewer content={content} />
-      </Box>
-    </Grommet>
-  )
-}
+export function Default ({ content, contentLoadingState, dark }) {
+  const subjectSnapshot = SubjectFactory.build({
+    content,
+    contentLoadingState,
+    locations: [
+      { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
```

Comment:
You might be able to use [sinon's fake server](https://sinonjs.org/releases/v14/fake-xhr-and-server/) to mock a response for this URL.

Reply:
Thanks, sinon's fake server looks pretty helpful! I can think of a few places that might help. I'll note for follow-up with other DigiLeap related refactors planned.